src/Cli/dotnet/commands/dotnet-sln/list/SlnListParser.cs

```diff
@@ -27,6 +32,7 @@ private static CliCommand ConstructCommand()
         CliCommand command = new("list", LocalizableStrings.ListAppFullName);
 
         command.Options.Add(SolutionFolderOption);
+        command.Options.Add(OutputFormatOption);
```

Comment:
What happens if the user passes in something invalid? And is this case-sensitive or insensitive?